### PR TITLE
Javascript-I Hw

### PR DESCRIPTION
**Pull request by IllSmithDa**

### CODE REVIEW

**Comment by IllSmithDa** [comment]
Samuel Kim - JavaScript Hw

File: src/callbacks.js [hunk 1]
```diff
@@ -69,12 +104,38 @@ contains(foods, 'ribeye', (result) => {
...
+    newObject[val]++;
+  });
+
+  for (let i = 0; i < arr.length; i++) {
+    const num = arr[i];
+    if (newObject[num] % 2 === 1) {
+      newArray.push(num)
+    }
+  }
+  cb(newArray);
```
Comment (sperrye) [inline src/callbacks.js:123]: You'll need to initialize newArray before you can push to it

File: src/callbacks.js [hunk 2]
```diff
@@ -69,12 +104,38 @@ contains(foods, 'ribeye', (result) => {
...
   console.log(`foods with duplicates removed: ${uniqueFoods}`);
 });
 
+
+
 // Write a function called forEach that iterates over the provided array and passes the value and index into the callback.
 // code here
+const forEach = (arr, cb) => {
+  arr.forEach((val, index) => {
+    cb(val, index);
```
Comment (sperrye) [inline src/callbacks.js:136]: Try re-writing this without using .forEach()

File: src/objects.js [hunk 3]
```diff
@@ -5,22 +5,43 @@ const keys = (obj) => {
   // Retrieve all the names of the object's properties.
   // Return the keys as strings in an array.
   // Based on http://underscorejs.org/#keys
+  const newArray = [];
+  Object.keys(obj).forEach((key) => {
```
Comment (sperrye) [inline src/objects.js:9]: Object.keys(obj) returns an array of keys on obj.

**Review by sperrye — COMMENTED** [review]
Great job

**Pull request closed without merging**